kernels/streamer_alu/Makefile

```diff
@@ -0,0 +1,36 @@
+# Courtesy of Federico Ficarelli
+
+.DEFAULT_GOAL := all
+
+include ../../runtime/snax-alu.rules
+include ../../runtime/Makefile.rules
+
+TESTS =
+TESTS += streamer_mult.x
```

Comment:
wait, this works even though there are no files named streamer_mult.x?

Reply:
yes through some makefile magic, streamer_mult.mlir is added in the next pr